client/rpc-core/src/eth.rs

```diff
@@ -85,6 +85,13 @@ pub trait EthApi {
 		number: BlockNumber,
 	) -> RpcResult<Option<U256>>;
 
+	/// Returns the receipts of a block by number or hash.
+	#[method(name = "eth_getBlockReceipts")]
+	async fn block_transaction_receipts(
+		&self,
+		number: BlockNumber,
+	) -> RpcResult<Vec<Option<Receipt>>>;
```

Comment:
```suggestion
	) -> RpcResult<Option<Vec<Receipt>>>;
```

Reply:
Use the same return type with the `transaction_receipt` to avoid another type conversion. The current way looks Okay to me.

Reply:
The semantics of return value is definitely problematic.
If this block number doesn't exist, shall we should return an empty array or `null`?

Reply:
Oh, I got you. Fixed